maas/client/facade.py

```diff
@@ -139,7 +139,8 @@ def machines(origin):
             "allocate": origin.Machines.allocate,
             "get": origin.Machine.read,
             "list": origin.Machines.read,
-            "get_power_parameters": origin.Machines.get_power_parameters,
+            "get_power_parameters_for":
```

Comment:
No, Machines.get_power_parameters_for gets a list of power parameters for a list of system_ids. Individual machines also have a get_power_parameters which returns just that machines power parameters. That's not mentioned in the facade because it's on the Machine object itself